packages/tools/devtools/devtools-browser-extension/src/BackgroundConnection.ts

```diff
@@ -54,15 +67,42 @@ import {
  */
 export class BackgroundConnection
 	extends TypedEventEmitter<IMessageRelayEvents>
-	implements IMessageRelay
+	implements IMessageRelay, IDisposable
```

Comment:
I see the "infrastructure" for `IDisposable` but no call to `dispose()`. Is that something the browser would do for us automatically?

Reply:
I actually don't need this, and meant to revert. Will do so now.